pkg/local_object_storage/blobstor/blobovnicza.go

```diff
@@ -262,7 +260,7 @@ func (b *blobovniczas) delete(prm *DeleteSmallPrm) (res *DeleteSmallRes, err err
 			return nil, err
 		}
 
-		return b.deleteObject(blz, bPrm)
+		return b.deleteObject(blz, bPrm, prm.addr)
```

Comment:
why not just add getter to delete parameter `bPrm`?

as I see you put the same address twice: one pure and one wrapped in `bPrm`

Reply:
Because we have access to address on this level so pull it from the lower is redundant

Reply:
still do not get the idea of wrapping and passing something and just passing it at the same time

but ok